components/_project.scss

```diff
@@ -72,6 +72,76 @@
         letter-spacing: 0.03em;
         margin: 10px 0;
       }
+
+      .modal-container {
+        position: fixed;
+        top: 0;
+        left: 0;
+        width: 100vw;
+        height: 100vh;
+        opacity: 0;
+        pointer-events: none;
+        backdrop-filter: blur(6px);
+        overflow: auto;
+
+        &.show-modal {
+          opacity: 1;
+          pointer-events: auto;
+        }
+
+        .modal {
+          width: 100%;
+          height: 120%;
+          display: grid;
+          place-items: center;
+
+          .project {
+            width: 87.5%;
+            margin: auto;
+
+            .primary-text {
+              width: 100%;
+
+              .modal-header {
+                display: flex;
+                justify-content: space-between;
+                align-items: center;
+
+                .close-modal {
+                  background-image: url(../images/svg/cancel-close-delete.svg);
+                  background-size: 100%;
+                  background-color: transparent;
+                  border: none;
+                  width: 16px;
+                  height: 16px;
+                  cursor: pointer;
+                  margin-right: 10px;
+                }
+              }
+            }
+
+            .project-text {
+              .modal-buttons {
+                display: flex;
+                justify-content: space-between;
+
```

Comment:
- [ ] The Languages in your modal are flex but the size in the desktop for them is too small, you can adjust the size of the modal and the text to give space to your languages.

- Your design.
![kkkk](https://user-images.githubusercontent.com/61562626/140963192-51877c3e-8a57-43e2-b742-5b72f4541144.jpeg)

- Figma design

![fffff](https://user-images.githubusercontent.com/61562626/140963460-78c91f04-810d-459e-bae5-8719b5b8f2ec.jpeg)

Reply:
I have made the changes, but  when went to the dashboard to submit a review. Instead of a Required Changes button, I see the yellow submit button.